schema/errors.libsonnet

```diff
@@ -0,0 +1,76 @@
+//   Copyright 2018 The Linux Foundation
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+
+local JSV7 = 'jsonschemaV7';
+local JSV7Schema = 'http://json-schema.org/draft-07/schema#';
+
+// Descriptions for schema objects. Separated out to make code more readable.
+local d = {
+  errors: 'This property contains a list of errors messages returned from the server due to an invalid request.',
+};
+
+// Error Schema
+local errors(output=JSV7) = {
+
+  local code = {
+    type: 'string',
+    enum: [
+      'BLOB_UNKNOWN',
```

Comment:
This is why I don't like this stuff. What if we add a new error code? This validator fails if it encounters that. This is not an enum necessarily. There are set of known values and unknown future values.

Reply:
There are multiple ways to handle this in JSONSchema using `anyOf` or `oneOf`.